anesthetic/plotting/_matplotlib/hist.py

```diff
@@ -44,6 +52,77 @@ def _calculate_bins(self, data):
         )
         return bins
 
+    def _make_plot(self):
+        if version.parse(pd.__version__) >= version.parse("2.0.0"):
+            super._make_plot()
+            return
```

Comment:
This can be ignored for coverage purposes by putting in: at each relevant clause
`def _make_plot(self):  # pragma: no cover`

since it's effectively covered by the pandas test suite.